### DISCUSSION TITLE
Position of the cursor line while scrolling

### DISCUSSION THREAD

**Discussion by sakkamade** [atom/atom#22429] [Q&A] [answered]
When I scroll the screen by cursor, the cursor line always keeps in two lines from top and bottom window's borders.

Can I increase the number of minimal lines between the cursor and the window border?

**Comment by sakkamade** [comment, marked as answer]
For goodness' sake, I have found it, the "scrolloff" package have solved the question of mine.
Thank you!

**Reply by ThatXliner** [reply]
There is a package for everything... unless there isn't. Then, there *will*.

**Reply by sakkamade** [reply]
Hopefully.
